lib/sdf-server/src/service/v2/approval_requirement_definition/new.rs

```diff
@@ -46,14 +44,44 @@ pub async fn new(
     }
 
     // TODO(nick): add audit logs, posthog tracking and WsEvent(s).
-    ApprovalRequirement::new_definition(
+    let approval_requirement_definition_id = ApprovalRequirement::new_definition(
         &ctx,
         request.entity_id,
         1, // TODO(nick): allow users to change the minimum approvers count
         approvers,
     )
     .await?;
 
+    let entity_kind = EntityKind::get_entity_kind_for_id(&ctx, request.entity_id).await?;
+    let entity_name = EntityKind::get_entity_name_for_id(&ctx, request.entity_id).await?;
+    let title = format!(
+        "{entity_kind} - {}",
+        entity_name
+            .to_owned()
+            .unwrap_or(request.entity_id.to_string())
```

Comment:
```suggestion
            .unwrap_or_else(|| request.entity_id.to_string())
```

Mainly to avoid the `to_string()` unless we're actually going to use the String.